prac_06/used_cars.py

```diff
@@ -0,0 +1,28 @@
+"""CP1404/CP5632 Practical - Client code to use the Car class."""
+# Note that the import has a folder (module) in it.
+
+from prac_06.car import Car
+
+
+def main():
+    """Demo test code to show how to use car class.
```

Comment:
Needs a capital, "How to use a Car class."

Reply:
Cool, thanks for pointing that out. will fix it